lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -44,45 +44,17 @@ using namespace llvm;
 
 namespace cling {
 
-IncrementalExecutor::IncrementalExecutor(clang::DiagnosticsEngine& diags,
-                                         const clang::CompilerInstance& CI):
-  m_externalIncrementalExecutor(nullptr),
-  m_CurrentAtExitModule(0)
-#if 0
-  : m_Diags(diags)
-#endif
-{
-
-  // MSVC doesn't support m_AtExitFuncsSpinLock=ATOMIC_FLAG_INIT; in the class definition
-  std::atomic_flag_clear( &m_AtExitFuncsSpinLock );
-
-  // No need to protect this access of m_AtExitFuncs, since nobody
-  // can use this object yet.
-  m_AtExitFuncs.reserve(256);
+namespace {
 
-  std::unique_ptr<TargetMachine>
-    TM(CreateHostTargetMachine(CI.getCodeGenOpts()));
-  m_BackendPasses.reset(new BackendPasses(CI.getCodeGenOpts(),
-                                          CI.getTargetOpts(),
-                                          CI.getLangOpts(),
-                                          *TM));
-  m_JIT.reset(new IncrementalJIT(*this, std::move(TM)));
-}
-
-// Keep in source: ~unique_ptr<ClingJIT> needs ClingJIT
-IncrementalExecutor::~IncrementalExecutor() {}
-
-std::unique_ptr<TargetMachine>
-  IncrementalExecutor::CreateHostTargetMachine(const
-                                           clang::CodeGenOptions& CGOpt) const {
-  // TODO: make this configurable.
+static std::unique_ptr<TargetMachine>
+CreateHostTargetMachine(const clang::CodeGenOptions& CGOpt, unsigned Fmt = 0) {
```

Comment:
Could you document the use of `Fmt` for future reference. Please add some documentation explaining that we default to Triple unless Fmt is specified.